BinaryEncoding.md

```diff
@@ -253,9 +253,9 @@ The import section declares all imports that will be used in the module.
 | Field | Type | Description |
 | ----- | ---- | ----------- |
 | module_len | `varuint32` | module string length |
-| module_str | `bytes` | module string of `module_len` bytes |
+| module_str | `bytes` | module name: `module_len` bytes holding valid UTF-8 string |
```

Comment:
This would be a valid UTF-8 byte sequence. A string is what you get after you decode.